src/frontend/apps/desk/src/pages/index.tsx

```diff
@@ -1,16 +1,36 @@
 import { useRouter as useNavigate } from 'next/navigation';
 import { useEffect } from 'react';
 
+import { useAuthStore } from '@/core/auth';
 import { useConfigStore } from '@/core/config';
 import { NextPageWithLayout } from '@/types/next';
 
 const Page: NextPageWithLayout = () => {
   const { config } = useConfigStore();
+  const { userData } = useAuthStore();
   const router = useNavigate();
 
   useEffect(() => {
+    if (!userData) {
+      router.push('/authenticate/');
+      return;
+    }
```

Comment:
The `useAuthStore` should already do that, not sure it's ok to do this again here, at least not this way?